tests/test_put_sentence.rs

```diff
@@ -148,3 +148,47 @@ fn test_put_sentence_language_that_does_not_exist_returns_404() {
         StatusCode::NotFound,
     );
 }
+
+#[test]
+fn test_put_sentence_structure_returns_200() {
+
+    let connection = db::get_connection();
+    db::clear(&connection);
+
+    let sentence_uuid = uuid::Uuid::new_v4();
+    let sentence_text = "This is one sentence";
+    let sentence_iso639_3 = "eng";
+    db::insert_sentence(
+        &connection,
+        &sentence_uuid,
+        &sentence_text,
+        &sentence_iso639_3,
+    );
+
+    let client = reqwest::Client::new();
+
+    let url = format!(
+        "{}/sentences/{}/structure",
+        tests_commons::SERVICE_URL,
+        sentence_uuid,
+    );
+    let modified_structure = "<tag>value</tag>";
```

Comment:
put directly  something like this `<sentence>This is one sentence</sentence>` because in the future we will certainly add more check that the xml schema is right, that the text content of the xml is the same as the one of the raw text etc.